river/proba/gaussian.py

```diff
@@ -214,7 +214,11 @@ def __init__(self, seed=None):
         super().__init__(seed)
         self._var = covariance.EmpiricalCovariance(ddof=1)
 
-    # TODO: add method _from_state to initialize model (for warm starting)
+    @classmethod
+    def _from_state(cls, n, mean, cov, ddof):
```

Comment:
The `seed` parameter will always be `None` in this case. It should be optionally passed as a parameter to ensure reproducibility.